### DISCUSSION TITLE
Ability to display where the form is used, or if not used

### DISCUSSION THREAD

**Discussion by yingban** [solspace/craft-freeform#1617] [Feature Requests]
Hi,

Would it be possible to add in the "form settings" panel, on which entry (with a link containing the value getCpEditUrl()) the form is used? 

Thank you!

**Comment by kjmartens** [comment]
Thanks for your feedback @yingban. We plan on adding support for this soon. 🙂

**Reply by yingban** [reply]
@kjmartens great ! thank you very much for the feature

**Comment by kjmartens** [comment]
This is now available in Freeform 5.7.2+ (on Craft 5) 🙂

**Comment by yingban** [comment]
@kjmartens Thank you very much!
